tt_metal/impl/dispatch/kernels/packet_queue.hpp

```diff
@@ -508,20 +529,22 @@ class packet_input_queue_state_t : public packet_queue_state_t {
         this->ptr_offset_mask = queue_size_words - 1;
         this->queue_size_mask = (queue_size_words << 1) - 1;
         this->curr_packet_valid = false;
-        this->reset_queue_local_rptr_sent();
-        this->reset_queue_local_rptr_cleared();
         this->reset_queue_local_wptr();
         this->reset_ready_flag();
     }
 
     inline uint32_t get_end_of_cmd() const {
-        return this->end_of_cmd;
+        return (uint32_t)this->end_of_cmd;
```

Comment:
Could we return a uint16 here and make `forward_data_from_input` accept a uint16?

Reply:
Fixed.